go.mod

```diff
@@ -8,7 +8,8 @@ go 1.21
 
 require (
 	dario.cat/mergo v1.0.0
-	github.com/aws/aws-sdk-go-v2 v1.24.0
+	github.com/aws/aws-sdk-go v1.49.2
+	github.com/aws/aws-sdk-go-v2 v1.24.1
```

Comment:
These bumps are coming from the native provider's upbound fork dependency versions.